frappe/utils/data.py

```diff
@@ -23,867 +23,881 @@
 
 # datetime functions
 def getdate(string_date=None):
-	"""
-		 Coverts string date (yyyy-mm-dd) to datetime.date object
-	"""
-	if not string_date:
-		return get_datetime().date()
+    """
```

Comment:
the project uses tabs!

Reply:
I'll switch to tabs. Pylint doesn't like it.